modules/migration/milestone.go

```diff
@@ -9,11 +9,11 @@ import "time"
 
 // Milestone defines a standard milestone
 type Milestone struct {
-	Title       string
-	Description string
-	Deadline    *time.Time
-	Created     time.Time
-	Updated     *time.Time
-	Closed      *time.Time
-	State       string // open, closed
+	Title       string     `json:"title"`
+	Description string     `json:"description"`
+	Deadline    *time.Time `json:"deadline"`
+	Created     time.Time  `json:"created"`
+	Updated     *time.Time `json:"updated"`
+	Closed      *time.Time `json:"closed"`
+	State       string     `json:"state"`
```

Comment:
Ah, the comment should be kept.

Reply:
@lunny good catch! The comment was restored.

---
[source](https://lab.fedeproxy.eu/forgefriends/forgefriends/-/merge_requests/32#note_889)